lib/plugin/stepTimeout.js

```diff
@@ -85,6 +85,6 @@ module.exports = (config) => {
       }
     }
     stepTimeout = stepTimeout === undefined ? config.timeout : stepTimeout;
-    step.totalTimeout = stepTimeout * 1000;
+    step.setTotalTimeout(stepTimeout * 1000, config.force ? 1 : 3);
```

Comment:
what does `config.force` do here? 
I thought it will make global step timeouts more prioritized over local step timeouts. Does it?

Reply:
`force` makes config values to override values set from code `I.limitTime(t)`

Reply:
Ok, got it! But the name is very unclear... How about a more verbose name?

```
ignoreStepLimits: true
```